doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -3184,25 +3183,25 @@ In a rewrite tactic of the form:
    rewrite occ_switch [term1]term2.
 
 ``term1`` is the explicit rewrite redex and ``term2`` is the rewrite rule.
-This execution of this tactic unfolds as follows:
+This execution of this tactic unfolds as follows.
 
 
 + First ``term1`` and ``term2`` are βι normalized. Then ``term2``
   is put in head
   normal form if the Leibniz equality constructor ``eq`` is not the head
   symbol. This may involve ζ reductions.
-+ Then, the matching algorithm (see section :ref:`abbreviations_ssr`)
++ Then, the matching algorithm (see Section :ref:`abbreviations_ssr`)
   determines the
   first subterm of the goal matching the rewrite pattern. The rewrite
   pattern is given by ``term1``, if an explicit redex pattern switch is
   provided, or by the type of ``term2`` otherwise. However, matching skips
   over matches that would lead to trivial rewrites. All the occurrences
   of this subterm in the goal are candidates for rewriting.
-+ Then only the occurrences coded by :token:`occ_switch` (see again section
++ Then only the occurrences coded by :token:`occ_switch` (see again Section
   :ref:`abbreviations_ssr`) are finally selected for rewriting.
-+ The left hand side of ``term2`` is unified with the subterm found by
++ The left-hand side of ``term2`` is unified with the subterm found by
```

Comment:
```suggestion
+ The left hand side of ``term2`` is unified with the subterm found by
```

Dash is not used for "left hand"

Reply:
I let you decide, but the hyphen should apparently be there. See for instance [https://ell.stackexchange.com/questions/169334/left-handside-left-hand-side-left-hand-side](url)

Reply:
I can't access that webpage, either, but after checking a bit more, it's probably more common with a hyphen.